sdk/aqueduct/aqueduct_client.py

```diff
@@ -454,18 +454,16 @@ def delete_flow(
         for integration in resp.saved_object_deletion_results:
             failed_for_integration = []
             for obj in resp.saved_object_deletion_results[integration]:
-                if not obj.succeeded:
+                if not obj.exec_state.status == ExecutionStatus.SUCCEEDED:
                     failed_for_integration.append(obj)
             if len(failed_for_integration) > 0:
                 failed_deletions[str(integration)] = [
-                    res.__dict__ for res in failed_for_integration
+                    json.loads(res.json()) for res in failed_for_integration
```

Comment:
[class].json() gives me the json as a string which gets escaped when I do `json.dumps`. However, I cannot serialize it directly with `json.dumps` so I serialize with `.json()` then `json.loads()` to get the format that is serializable by `json.dumps`.